platform/ext/target/nordic_nrf/Native_Driver/spu.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __SPU_H__
+#define __SPU_H__
+
+#include <stddef.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+/**
+ * \brief Reset all memory regions to being Secure
+ *
+ * Reset all (Flash or SRAM) memory regions to being Secure
+ * and have default (i.e. Read-Write-Execute allow) access policy
+ *
+ * Note: region lock not enforced, to allow modifying the applied configuration.
```

Comment:
Nit: This line is formatted differently for the three functions ("Note:", "enforced" vs "applied"). The note could be a dox `\note`.

I also suggest adding something like "until the next reset" for people not familiar with the locking functionality.

Reply:
both suggestions are good, actually, I also thought of having the "until next reset cycle"
Let me apply this

Reply:
done